2022-08-03 18:00:41,707 INFO    MainThread:22252 [internal.py:wandb_internal():90] W&B internal server running at pid: 22252, started at: 2022-08-03 18:00:41.707255
2022-08-03 18:00:41,709 INFO    WriterThread:22252 [datastore.py:open_for_write():75] open: D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\run-mvzyeprd.wandb
2022-08-03 18:00:41,719 DEBUG   SenderThread:22252 [sender.py:send():236] send: header
2022-08-03 18:00:41,719 DEBUG   SenderThread:22252 [sender.py:send():236] send: run
2022-08-03 18:00:42,281 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: check_version
2022-08-03 18:00:42,284 INFO    SenderThread:22252 [dir_watcher.py:__init__():216] watching files in: D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\files
2022-08-03 18:00:42,286 INFO    SenderThread:22252 [sender.py:_start_run_threads():815] run started: mvzyeprd with start time 1659520840
2022-08-03 18:00:42,286 DEBUG   SenderThread:22252 [sender.py:send():236] send: summary
2022-08-03 18:00:42,286 INFO    SenderThread:22252 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2022-08-03 18:00:42,287 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: check_version
2022-08-03 18:00:43,287 INFO    Thread-11 :22252 [dir_watcher.py:_on_file_created():269] file/dir created: D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\files\wandb-summary.json
2022-08-03 18:00:44,776 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: run_start
2022-08-03 18:00:45,237 DEBUG   HandlerThread:22252 [meta.py:__init__():35] meta init
2022-08-03 18:00:45,237 DEBUG   HandlerThread:22252 [meta.py:__init__():49] meta init done
2022-08-03 18:00:45,237 DEBUG   HandlerThread:22252 [meta.py:probe():209] probe
2022-08-03 18:00:45,276 DEBUG   HandlerThread:22252 [git.py:repo():29] git repository is invalid
2022-08-03 18:00:45,276 DEBUG   HandlerThread:22252 [meta.py:_save_pip():53] save pip
2022-08-03 18:00:45,289 DEBUG   HandlerThread:22252 [meta.py:_save_pip():67] save pip done
2022-08-03 18:00:45,289 DEBUG   HandlerThread:22252 [meta.py:_save_conda():74] save conda
2022-08-03 18:00:45,312 INFO    Thread-11 :22252 [dir_watcher.py:_on_file_created():269] file/dir created: D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\files\requirements.txt
2022-08-03 18:00:45,312 INFO    Thread-11 :22252 [dir_watcher.py:_on_file_created():269] file/dir created: D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\files\conda-environment.yaml
2022-08-03 18:00:50,375 INFO    Thread-11 :22252 [dir_watcher.py:_on_file_created():269] file/dir created: D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\files\output.log
2022-08-03 18:00:51,877 DEBUG   HandlerThread:22252 [meta.py:_save_conda():84] save conda done
2022-08-03 18:00:51,877 DEBUG   HandlerThread:22252 [meta.py:probe():247] probe done
2022-08-03 18:00:52,027 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: stop_status
2022-08-03 18:00:52,028 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: stop_status
2022-08-03 18:00:52,393 INFO    Thread-11 :22252 [dir_watcher.py:_on_file_modified():286] file/dir modified: D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\files\output.log
2022-08-03 18:00:52,393 INFO    Thread-11 :22252 [dir_watcher.py:_on_file_modified():286] file/dir modified: D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\files\conda-environment.yaml
2022-08-03 18:00:52,394 INFO    Thread-11 :22252 [dir_watcher.py:_on_file_created():269] file/dir created: D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\files\wandb-metadata.json
2022-08-03 18:00:55,679 DEBUG   SenderThread:22252 [sender.py:send():236] send: telemetry
2022-08-03 18:00:55,680 DEBUG   SenderThread:22252 [sender.py:send():236] send: telemetry
2022-08-03 18:00:55,680 DEBUG   SenderThread:22252 [sender.py:send():236] send: config
2022-08-03 18:00:55,681 DEBUG   SenderThread:22252 [sender.py:send():236] send: config
2022-08-03 18:00:55,681 DEBUG   SenderThread:22252 [sender.py:send():236] send: config
2022-08-03 18:00:55,681 DEBUG   SenderThread:22252 [sender.py:send():236] send: config
2022-08-03 18:00:55,681 DEBUG   SenderThread:22252 [sender.py:send():236] send: config
2022-08-03 18:00:55,681 DEBUG   SenderThread:22252 [sender.py:send():236] send: config
2022-08-03 18:00:55,682 DEBUG   SenderThread:22252 [sender.py:send():236] send: config
2022-08-03 18:00:55,682 DEBUG   SenderThread:22252 [sender.py:send():236] send: config
2022-08-03 18:00:55,682 DEBUG   SenderThread:22252 [sender.py:send():236] send: config
2022-08-03 18:00:55,682 DEBUG   SenderThread:22252 [sender.py:send():236] send: config
2022-08-03 18:00:55,682 DEBUG   SenderThread:22252 [sender.py:send():236] send: telemetry
2022-08-03 18:00:55,683 DEBUG   SenderThread:22252 [sender.py:send():236] send: files
2022-08-03 18:00:55,684 INFO    SenderThread:22252 [sender.py:_save_file():952] saving file wandb-metadata.json with policy now
2022-08-03 18:00:56,852 INFO    Thread-15 :22252 [upload_job.py:push():138] Uploaded file C:\Users\Yy\AppData\Local\Temp\tmph3oo6jdfwandb\28a29gs5-wandb-metadata.json
2022-08-03 18:01:09,822 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: stop_status
2022-08-03 18:01:09,822 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: stop_status
2022-08-03 18:01:15,515 INFO    Thread-11 :22252 [dir_watcher.py:_on_file_modified():286] file/dir modified: D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\files\config.yaml
2022-08-03 18:01:22,116 DEBUG   SenderThread:22252 [sender.py:send():236] send: stats
2022-08-03 18:01:25,584 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: stop_status
2022-08-03 18:01:25,584 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: stop_status
2022-08-03 18:01:41,719 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: stop_status
2022-08-03 18:01:41,719 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: stop_status
2022-08-03 18:01:57,205 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: stop_status
2022-08-03 18:01:57,205 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: stop_status
2022-08-03 18:01:57,998 DEBUG   SenderThread:22252 [sender.py:send():236] send: stats
2022-08-03 18:02:12,757 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: stop_status
2022-08-03 18:02:12,758 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: stop_status
2022-08-03 18:02:26,046 DEBUG   SenderThread:22252 [sender.py:send():236] send: telemetry
2022-08-03 18:02:26,242 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:02:26,242 DEBUG   SenderThread:22252 [sender.py:send():236] send: exit
2022-08-03 18:02:26,243 INFO    SenderThread:22252 [sender.py:send_exit():372] handling exit code: 1
2022-08-03 18:02:26,243 INFO    SenderThread:22252 [sender.py:send_exit():374] handling runtime: 101
2022-08-03 18:02:26,244 INFO    SenderThread:22252 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2022-08-03 18:02:26,245 INFO    SenderThread:22252 [sender.py:send_exit():380] send defer
2022-08-03 18:02:26,245 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:02:26,245 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: defer
2022-08-03 18:02:26,246 INFO    HandlerThread:22252 [handler.py:handle_request_defer():163] handle defer: 0
2022-08-03 18:02:26,246 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: defer
2022-08-03 18:02:26,246 INFO    SenderThread:22252 [sender.py:send_request_defer():389] handle sender defer: 0
2022-08-03 18:02:26,246 INFO    SenderThread:22252 [sender.py:transition_state():393] send defer: 1
2022-08-03 18:02:26,246 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: defer
2022-08-03 18:02:26,247 INFO    HandlerThread:22252 [handler.py:handle_request_defer():163] handle defer: 1
2022-08-03 18:02:26,254 WARNING FileStreamThread:22252 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/vs74/Expand-ai-problem-2/mvzyeprd/file_stream (Caused by ProxyError('Cannot connect to proxy.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001BC49B150A0>>, args: ('https://api.wandb.ai/files/vs74/Expand-ai-problem-2/mvzyeprd/file_stream',), kwargs: {'json': {'files': {'output.log': {'offset': 2, 'content': ['ERROR 2022-08-03T10:02:26.241787 Traceback (most recent call last):\n', 'ERROR 2022-08-03T10:02:26.241787   File "D:/my_code_2/my_code/tutorial-MixMatch-master/MixMatch_pycharm_0801.py", line 927, in <module>\n', 'ERROR 2022-08-03T10:02:26.241787     main()\n', 'ERROR 2022-08-03T10:02:26.241787   File "D:/my_code_2/my_code/tutorial-MixMatch-master/MixMatch_pycharm_0801.py", line 923, in main\n', 'ERROR 2022-08-03T10:02:26.241787     train_model(model, labeled_trainloader, unlabeled_trainloader, epochs, log_freq)\n', 'ERROR 2022-08-03T10:02:26.241787   File "D:/my_code_2/my_code/tutorial-MixMatch-master/MixMatch_pycharm_0801.py", line 849, in train_model\n', 'ERROR 2022-08-03T10:02:26.241787     train_loss, train_loss_x, train_loss_u = train(labeled_trainloader, unlabeled_trainloader, model, epoch=epoch,\n', 'ERROR 2022-08-03T10:02:26.241787   File "D:/my_code_2/my_code/tutorial-MixMatch-master/MixMatch_pycharm_0801.py", line 726, in train\n', 'ERROR 2022-08-03T10:02:26.241787     logits.append(model(input))\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\module.py", line 1120, in _call_impl\n', 'ERROR 2022-08-03T10:02:26.241787     result = forward_call(*input, **kwargs)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torchvision\\models\\resnet.py", line 249, in forward\n', 'ERROR 2022-08-03T10:02:26.241787     return self._forward_impl(x)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torchvision\\models\\resnet.py", line 238, in _forward_impl\n', 'ERROR 2022-08-03T10:02:26.241787     x = self.layer2(x)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\module.py", line 1102, in _call_impl\n', 'ERROR 2022-08-03T10:02:26.241787     return forward_call(*input, **kwargs)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\container.py", line 141, in forward\n', 'ERROR 2022-08-03T10:02:26.241787     input = module(input)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\module.py", line 1102, in _call_impl\n', 'ERROR 2022-08-03T10:02:26.241787     return forward_call(*input, **kwargs)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torchvision\\models\\resnet.py", line 124, in forward\n', 'ERROR 2022-08-03T10:02:26.241787     out = self.conv1(x)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\module.py", line 1102, in _call_impl\n', 'ERROR 2022-08-03T10:02:26.241787     return forward_call(*input, **kwargs)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\conv.py", line 446, in forward\n', 'ERROR 2022-08-03T10:02:26.241787     return self._conv_forward(input, self.weight, self.bias)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\conv.py", line 442, in _conv_forward\n', 'ERROR 2022-08-03T10:02:26.241787     return F.conv2d(input, weight, bias, self.stride,\n', 'ERROR 2022-08-03T10:02:26.241787 RuntimeError: [enforce fail at ..\\c10\\core\\CPUAllocator.cpp:76] data. DefaultCPUAllocator: not enough memory: you tried to allocate 13844480 bytes.\n']}}, 'dropped': 0}}
2022-08-03 18:02:26,276 INFO    Thread-11 :22252 [dir_watcher.py:_on_file_modified():286] file/dir modified: D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\files\wandb-summary.json
2022-08-03 18:02:26,277 INFO    Thread-11 :22252 [dir_watcher.py:_on_file_modified():286] file/dir modified: D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\files\output.log
2022-08-03 18:02:26,483 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: defer
2022-08-03 18:02:26,483 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:02:26,484 INFO    SenderThread:22252 [sender.py:send_request_defer():389] handle sender defer: 1
2022-08-03 18:02:26,484 INFO    SenderThread:22252 [sender.py:transition_state():393] send defer: 2
2022-08-03 18:02:26,484 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:02:26,485 DEBUG   SenderThread:22252 [sender.py:send():236] send: stats
2022-08-03 18:02:26,486 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: defer
2022-08-03 18:02:26,486 INFO    HandlerThread:22252 [handler.py:handle_request_defer():163] handle defer: 2
2022-08-03 18:02:26,486 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: defer
2022-08-03 18:02:26,486 INFO    SenderThread:22252 [sender.py:send_request_defer():389] handle sender defer: 2
2022-08-03 18:02:26,486 INFO    SenderThread:22252 [sender.py:transition_state():393] send defer: 3
2022-08-03 18:02:26,486 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: defer
2022-08-03 18:02:26,487 INFO    HandlerThread:22252 [handler.py:handle_request_defer():163] handle defer: 3
2022-08-03 18:02:26,487 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: defer
2022-08-03 18:02:26,487 INFO    SenderThread:22252 [sender.py:send_request_defer():389] handle sender defer: 3
2022-08-03 18:02:26,487 INFO    SenderThread:22252 [sender.py:transition_state():393] send defer: 4
2022-08-03 18:02:26,487 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: defer
2022-08-03 18:02:26,487 INFO    HandlerThread:22252 [handler.py:handle_request_defer():163] handle defer: 4
2022-08-03 18:02:26,488 DEBUG   SenderThread:22252 [sender.py:send():236] send: summary
2022-08-03 18:02:26,488 INFO    SenderThread:22252 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2022-08-03 18:02:26,489 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: defer
2022-08-03 18:02:26,489 INFO    SenderThread:22252 [sender.py:send_request_defer():389] handle sender defer: 4
2022-08-03 18:02:26,489 INFO    SenderThread:22252 [sender.py:transition_state():393] send defer: 5
2022-08-03 18:02:26,489 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: defer
2022-08-03 18:02:26,489 INFO    HandlerThread:22252 [handler.py:handle_request_defer():163] handle defer: 5
2022-08-03 18:02:26,489 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: defer
2022-08-03 18:02:26,489 INFO    SenderThread:22252 [sender.py:send_request_defer():389] handle sender defer: 5
2022-08-03 18:02:26,592 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:02:27,283 INFO    Thread-11 :22252 [dir_watcher.py:_on_file_modified():286] file/dir modified: D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\files\wandb-summary.json
2022-08-03 18:02:30,449 WARNING FileStreamThread:22252 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/vs74/Expand-ai-problem-2/mvzyeprd/file_stream (Caused by ProxyError('Cannot connect to proxy.', NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001BC4A084EE0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001BC49B150A0>>, args: ('https://api.wandb.ai/files/vs74/Expand-ai-problem-2/mvzyeprd/file_stream',), kwargs: {'json': {'files': {'output.log': {'offset': 2, 'content': ['ERROR 2022-08-03T10:02:26.241787 Traceback (most recent call last):\n', 'ERROR 2022-08-03T10:02:26.241787   File "D:/my_code_2/my_code/tutorial-MixMatch-master/MixMatch_pycharm_0801.py", line 927, in <module>\n', 'ERROR 2022-08-03T10:02:26.241787     main()\n', 'ERROR 2022-08-03T10:02:26.241787   File "D:/my_code_2/my_code/tutorial-MixMatch-master/MixMatch_pycharm_0801.py", line 923, in main\n', 'ERROR 2022-08-03T10:02:26.241787     train_model(model, labeled_trainloader, unlabeled_trainloader, epochs, log_freq)\n', 'ERROR 2022-08-03T10:02:26.241787   File "D:/my_code_2/my_code/tutorial-MixMatch-master/MixMatch_pycharm_0801.py", line 849, in train_model\n', 'ERROR 2022-08-03T10:02:26.241787     train_loss, train_loss_x, train_loss_u = train(labeled_trainloader, unlabeled_trainloader, model, epoch=epoch,\n', 'ERROR 2022-08-03T10:02:26.241787   File "D:/my_code_2/my_code/tutorial-MixMatch-master/MixMatch_pycharm_0801.py", line 726, in train\n', 'ERROR 2022-08-03T10:02:26.241787     logits.append(model(input))\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\module.py", line 1120, in _call_impl\n', 'ERROR 2022-08-03T10:02:26.241787     result = forward_call(*input, **kwargs)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torchvision\\models\\resnet.py", line 249, in forward\n', 'ERROR 2022-08-03T10:02:26.241787     return self._forward_impl(x)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torchvision\\models\\resnet.py", line 238, in _forward_impl\n', 'ERROR 2022-08-03T10:02:26.241787     x = self.layer2(x)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\module.py", line 1102, in _call_impl\n', 'ERROR 2022-08-03T10:02:26.241787     return forward_call(*input, **kwargs)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\container.py", line 141, in forward\n', 'ERROR 2022-08-03T10:02:26.241787     input = module(input)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\module.py", line 1102, in _call_impl\n', 'ERROR 2022-08-03T10:02:26.241787     return forward_call(*input, **kwargs)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torchvision\\models\\resnet.py", line 124, in forward\n', 'ERROR 2022-08-03T10:02:26.241787     out = self.conv1(x)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\module.py", line 1102, in _call_impl\n', 'ERROR 2022-08-03T10:02:26.241787     return forward_call(*input, **kwargs)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\conv.py", line 446, in forward\n', 'ERROR 2022-08-03T10:02:26.241787     return self._conv_forward(input, self.weight, self.bias)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\conv.py", line 442, in _conv_forward\n', 'ERROR 2022-08-03T10:02:26.241787     return F.conv2d(input, weight, bias, self.stride,\n', 'ERROR 2022-08-03T10:02:26.241787 RuntimeError: [enforce fail at ..\\c10\\core\\CPUAllocator.cpp:76] data. DefaultCPUAllocator: not enough memory: you tried to allocate 13844480 bytes.\n']}}, 'dropped': 0}}
2022-08-03 18:02:31,712 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:02:35,331 ERROR   SenderThread:22252 [retry.py:__call__():132] Retry attempt failed:
Traceback (most recent call last):
  File "C:\Users\Yy\miniconda3\lib\site-packages\urllib3\connectionpool.py", line 667, in urlopen
    self._prepare_proxy(conn)
  File "C:\Users\Yy\miniconda3\lib\site-packages\urllib3\connectionpool.py", line 932, in _prepare_proxy
    conn.connect()
  File "C:\Users\Yy\miniconda3\lib\site-packages\urllib3\connection.py", line 317, in connect
    self._tunnel()
  File "C:\Users\Yy\miniconda3\lib\http\client.py", line 900, in _tunnel
    (version, code, message) = response._read_status()
  File "C:\Users\Yy\miniconda3\lib\http\client.py", line 268, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
  File "C:\Users\Yy\miniconda3\lib\socket.py", line 669, in readinto
    return self._sock.recv_into(b)
ConnectionResetError: [WinError 10054] 远程主机强迫关闭了一个现有的连接。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Yy\miniconda3\lib\site-packages\requests\adapters.py", line 439, in send
    resp = conn.urlopen(
  File "C:\Users\Yy\miniconda3\lib\site-packages\urllib3\connectionpool.py", line 726, in urlopen
    retries = retries.increment(
  File "C:\Users\Yy\miniconda3\lib\site-packages\urllib3\util\retry.py", line 446, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /graphql (Caused by ProxyError('Cannot connect to proxy.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None)))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Yy\miniconda3\lib\site-packages\wandb\sdk\lib\retry.py", line 108, in __call__
    result = self._call_fn(*args, **kwargs)
  File "C:\Users\Yy\miniconda3\lib\site-packages\wandb\sdk\internal\internal_api.py", line 158, in execute
    return self.client.execute(*args, **kwargs)
  File "C:\Users\Yy\miniconda3\lib\site-packages\wandb\vendor\gql-0.2.0\wandb_gql\client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "C:\Users\Yy\miniconda3\lib\site-packages\wandb\vendor\gql-0.2.0\wandb_gql\client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "C:\Users\Yy\miniconda3\lib\site-packages\wandb\vendor\gql-0.2.0\wandb_gql\transport\requests.py", line 38, in execute
    request = requests.post(self.url, **post_args)
  File "C:\Users\Yy\miniconda3\lib\site-packages\requests\api.py", line 119, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "C:\Users\Yy\miniconda3\lib\site-packages\requests\api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "C:\Users\Yy\miniconda3\lib\site-packages\requests\sessions.py", line 530, in request
    resp = self.send(prep, **send_kwargs)
  File "C:\Users\Yy\miniconda3\lib\site-packages\requests\sessions.py", line 643, in send
    r = adapter.send(request, **kwargs)
  File "C:\Users\Yy\miniconda3\lib\site-packages\requests\adapters.py", line 510, in send
    raise ProxyError(e, request=request)
requests.exceptions.ProxyError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /graphql (Caused by ProxyError('Cannot connect to proxy.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None)))
2022-08-03 18:02:35,332 WARNING FileStreamThread:22252 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/vs74/Expand-ai-problem-2/mvzyeprd/file_stream (Caused by ProxyError('Cannot connect to proxy.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001BC49B150A0>>, args: ('https://api.wandb.ai/files/vs74/Expand-ai-problem-2/mvzyeprd/file_stream',), kwargs: {'json': {'files': {'output.log': {'offset': 2, 'content': ['ERROR 2022-08-03T10:02:26.241787 Traceback (most recent call last):\n', 'ERROR 2022-08-03T10:02:26.241787   File "D:/my_code_2/my_code/tutorial-MixMatch-master/MixMatch_pycharm_0801.py", line 927, in <module>\n', 'ERROR 2022-08-03T10:02:26.241787     main()\n', 'ERROR 2022-08-03T10:02:26.241787   File "D:/my_code_2/my_code/tutorial-MixMatch-master/MixMatch_pycharm_0801.py", line 923, in main\n', 'ERROR 2022-08-03T10:02:26.241787     train_model(model, labeled_trainloader, unlabeled_trainloader, epochs, log_freq)\n', 'ERROR 2022-08-03T10:02:26.241787   File "D:/my_code_2/my_code/tutorial-MixMatch-master/MixMatch_pycharm_0801.py", line 849, in train_model\n', 'ERROR 2022-08-03T10:02:26.241787     train_loss, train_loss_x, train_loss_u = train(labeled_trainloader, unlabeled_trainloader, model, epoch=epoch,\n', 'ERROR 2022-08-03T10:02:26.241787   File "D:/my_code_2/my_code/tutorial-MixMatch-master/MixMatch_pycharm_0801.py", line 726, in train\n', 'ERROR 2022-08-03T10:02:26.241787     logits.append(model(input))\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\module.py", line 1120, in _call_impl\n', 'ERROR 2022-08-03T10:02:26.241787     result = forward_call(*input, **kwargs)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torchvision\\models\\resnet.py", line 249, in forward\n', 'ERROR 2022-08-03T10:02:26.241787     return self._forward_impl(x)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torchvision\\models\\resnet.py", line 238, in _forward_impl\n', 'ERROR 2022-08-03T10:02:26.241787     x = self.layer2(x)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\module.py", line 1102, in _call_impl\n', 'ERROR 2022-08-03T10:02:26.241787     return forward_call(*input, **kwargs)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\container.py", line 141, in forward\n', 'ERROR 2022-08-03T10:02:26.241787     input = module(input)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\module.py", line 1102, in _call_impl\n', 'ERROR 2022-08-03T10:02:26.241787     return forward_call(*input, **kwargs)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torchvision\\models\\resnet.py", line 124, in forward\n', 'ERROR 2022-08-03T10:02:26.241787     out = self.conv1(x)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\module.py", line 1102, in _call_impl\n', 'ERROR 2022-08-03T10:02:26.241787     return forward_call(*input, **kwargs)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\conv.py", line 446, in forward\n', 'ERROR 2022-08-03T10:02:26.241787     return self._conv_forward(input, self.weight, self.bias)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\conv.py", line 442, in _conv_forward\n', 'ERROR 2022-08-03T10:02:26.241787     return F.conv2d(input, weight, bias, self.stride,\n', 'ERROR 2022-08-03T10:02:26.241787 RuntimeError: [enforce fail at ..\\c10\\core\\CPUAllocator.cpp:76] data. DefaultCPUAllocator: not enough memory: you tried to allocate 13844480 bytes.\n']}}, 'dropped': 0}}
2022-08-03 18:02:36,826 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:02:41,942 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:02:47,025 WARNING FileStreamThread:22252 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/vs74/Expand-ai-problem-2/mvzyeprd/file_stream (Caused by ProxyError('Cannot connect to proxy.', NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001BC4A078F40>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001BC49B150A0>>, args: ('https://api.wandb.ai/files/vs74/Expand-ai-problem-2/mvzyeprd/file_stream',), kwargs: {'json': {'files': {'output.log': {'offset': 2, 'content': ['ERROR 2022-08-03T10:02:26.241787 Traceback (most recent call last):\n', 'ERROR 2022-08-03T10:02:26.241787   File "D:/my_code_2/my_code/tutorial-MixMatch-master/MixMatch_pycharm_0801.py", line 927, in <module>\n', 'ERROR 2022-08-03T10:02:26.241787     main()\n', 'ERROR 2022-08-03T10:02:26.241787   File "D:/my_code_2/my_code/tutorial-MixMatch-master/MixMatch_pycharm_0801.py", line 923, in main\n', 'ERROR 2022-08-03T10:02:26.241787     train_model(model, labeled_trainloader, unlabeled_trainloader, epochs, log_freq)\n', 'ERROR 2022-08-03T10:02:26.241787   File "D:/my_code_2/my_code/tutorial-MixMatch-master/MixMatch_pycharm_0801.py", line 849, in train_model\n', 'ERROR 2022-08-03T10:02:26.241787     train_loss, train_loss_x, train_loss_u = train(labeled_trainloader, unlabeled_trainloader, model, epoch=epoch,\n', 'ERROR 2022-08-03T10:02:26.241787   File "D:/my_code_2/my_code/tutorial-MixMatch-master/MixMatch_pycharm_0801.py", line 726, in train\n', 'ERROR 2022-08-03T10:02:26.241787     logits.append(model(input))\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\module.py", line 1120, in _call_impl\n', 'ERROR 2022-08-03T10:02:26.241787     result = forward_call(*input, **kwargs)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torchvision\\models\\resnet.py", line 249, in forward\n', 'ERROR 2022-08-03T10:02:26.241787     return self._forward_impl(x)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torchvision\\models\\resnet.py", line 238, in _forward_impl\n', 'ERROR 2022-08-03T10:02:26.241787     x = self.layer2(x)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\module.py", line 1102, in _call_impl\n', 'ERROR 2022-08-03T10:02:26.241787     return forward_call(*input, **kwargs)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\container.py", line 141, in forward\n', 'ERROR 2022-08-03T10:02:26.241787     input = module(input)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\module.py", line 1102, in _call_impl\n', 'ERROR 2022-08-03T10:02:26.241787     return forward_call(*input, **kwargs)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torchvision\\models\\resnet.py", line 124, in forward\n', 'ERROR 2022-08-03T10:02:26.241787     out = self.conv1(x)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\module.py", line 1102, in _call_impl\n', 'ERROR 2022-08-03T10:02:26.241787     return forward_call(*input, **kwargs)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\conv.py", line 446, in forward\n', 'ERROR 2022-08-03T10:02:26.241787     return self._conv_forward(input, self.weight, self.bias)\n', 'ERROR 2022-08-03T10:02:26.241787   File "C:\\Users\\Yy\\miniconda3\\lib\\site-packages\\torch\\nn\\modules\\conv.py", line 442, in _conv_forward\n', 'ERROR 2022-08-03T10:02:26.241787     return F.conv2d(input, weight, bias, self.stride,\n', 'ERROR 2022-08-03T10:02:26.241787 RuntimeError: [enforce fail at ..\\c10\\core\\CPUAllocator.cpp:76] data. DefaultCPUAllocator: not enough memory: you tried to allocate 13844480 bytes.\n']}}, 'dropped': 0}}
2022-08-03 18:02:47,058 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:02:52,174 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:02:57,296 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:02,422 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:07,549 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:10,893 INFO    SenderThread:22252 [sender.py:transition_state():393] send defer: 6
2022-08-03 18:03:10,894 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:10,894 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:10,894 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:10,895 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:10,895 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:10,895 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:10,895 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:10,896 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:10,896 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:10,897 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: defer
2022-08-03 18:03:10,897 INFO    HandlerThread:22252 [handler.py:handle_request_defer():163] handle defer: 6
2022-08-03 18:03:10,898 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: defer
2022-08-03 18:03:10,898 INFO    SenderThread:22252 [sender.py:send_request_defer():389] handle sender defer: 6
2022-08-03 18:03:10,898 INFO    SenderThread:22252 [dir_watcher.py:finish():353] shutting down directory watcher
2022-08-03 18:03:11,011 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:11,655 INFO    SenderThread:22252 [dir_watcher.py:_on_file_modified():286] file/dir modified: D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\files\config.yaml
2022-08-03 18:03:11,655 INFO    SenderThread:22252 [dir_watcher.py:finish():383] scan: D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\files
2022-08-03 18:03:11,655 INFO    SenderThread:22252 [dir_watcher.py:finish():397] scan save: D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\files\conda-environment.yaml conda-environment.yaml
2022-08-03 18:03:11,656 INFO    SenderThread:22252 [dir_watcher.py:finish():397] scan save: D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\files\config.yaml config.yaml
2022-08-03 18:03:11,656 INFO    SenderThread:22252 [dir_watcher.py:finish():397] scan save: D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\files\output.log output.log
2022-08-03 18:03:11,660 INFO    SenderThread:22252 [dir_watcher.py:finish():397] scan save: D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\files\requirements.txt requirements.txt
2022-08-03 18:03:11,669 INFO    SenderThread:22252 [dir_watcher.py:finish():397] scan save: D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\files\wandb-metadata.json wandb-metadata.json
2022-08-03 18:03:11,669 INFO    SenderThread:22252 [dir_watcher.py:finish():397] scan save: D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\files\wandb-summary.json wandb-summary.json
2022-08-03 18:03:11,670 INFO    SenderThread:22252 [sender.py:transition_state():393] send defer: 7
2022-08-03 18:03:11,671 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:11,679 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: defer
2022-08-03 18:03:11,679 INFO    HandlerThread:22252 [handler.py:handle_request_defer():163] handle defer: 7
2022-08-03 18:03:11,680 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: defer
2022-08-03 18:03:11,680 INFO    SenderThread:22252 [sender.py:send_request_defer():389] handle sender defer: 7
2022-08-03 18:03:11,680 INFO    SenderThread:22252 [file_pusher.py:finish():171] shutting down file pusher
2022-08-03 18:03:11,779 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:11,779 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:11,890 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:11,890 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:11,999 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:12,000 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:12,111 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:12,111 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:12,218 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:12,218 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:12,326 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:12,326 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:12,436 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:12,437 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:12,544 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:12,545 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:12,653 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:12,654 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:12,763 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:12,763 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:12,872 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:12,873 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:12,981 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:12,981 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:13,091 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:13,091 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:13,201 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:13,202 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:13,311 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:13,312 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:13,422 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:13,423 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:13,533 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:13,534 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:13,645 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:13,645 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:13,755 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:13,755 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:13,866 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:13,866 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:13,976 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:13,976 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:14,088 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:14,088 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:14,197 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:14,197 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:14,306 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:14,307 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:14,417 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:14,417 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:14,528 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:14,528 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:14,637 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:14,638 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:14,746 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:14,747 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:14,857 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:14,858 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:14,965 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:14,966 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:14,971 INFO    Thread-16 :22252 [upload_job.py:push():138] Uploaded file D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\files\conda-environment.yaml
2022-08-03 18:03:15,024 INFO    Thread-17 :22252 [upload_job.py:push():138] Uploaded file D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\files\config.yaml
2022-08-03 18:03:15,031 INFO    Thread-19 :22252 [upload_job.py:push():138] Uploaded file D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\files\requirements.txt
2022-08-03 18:03:15,032 INFO    Thread-18 :22252 [upload_job.py:push():138] Uploaded file D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\files\output.log
2022-08-03 18:03:15,040 INFO    Thread-20 :22252 [upload_job.py:push():138] Uploaded file D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\files\wandb-summary.json
2022-08-03 18:03:15,073 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:15,073 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:15,183 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:15,183 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:15,245 INFO    Thread-10 :22252 [sender.py:transition_state():393] send defer: 8
2022-08-03 18:03:15,246 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: defer
2022-08-03 18:03:15,246 INFO    HandlerThread:22252 [handler.py:handle_request_defer():163] handle defer: 8
2022-08-03 18:03:15,246 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: defer
2022-08-03 18:03:15,246 INFO    SenderThread:22252 [sender.py:send_request_defer():389] handle sender defer: 8
2022-08-03 18:03:15,293 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:15,646 INFO    SenderThread:22252 [sender.py:transition_state():393] send defer: 9
2022-08-03 18:03:15,646 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:15,647 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: defer
2022-08-03 18:03:15,648 INFO    HandlerThread:22252 [handler.py:handle_request_defer():163] handle defer: 9
2022-08-03 18:03:15,648 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: defer
2022-08-03 18:03:15,648 INFO    SenderThread:22252 [sender.py:send_request_defer():389] handle sender defer: 9
2022-08-03 18:03:15,649 INFO    SenderThread:22252 [sender.py:transition_state():393] send defer: 10
2022-08-03 18:03:15,650 DEBUG   SenderThread:22252 [sender.py:send():236] send: final
2022-08-03 18:03:15,651 DEBUG   SenderThread:22252 [sender.py:send():236] send: footer
2022-08-03 18:03:15,652 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: defer
2022-08-03 18:03:15,652 INFO    HandlerThread:22252 [handler.py:handle_request_defer():163] handle defer: 10
2022-08-03 18:03:15,652 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: defer
2022-08-03 18:03:15,653 INFO    SenderThread:22252 [sender.py:send_request_defer():389] handle sender defer: 10
2022-08-03 18:03:15,765 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: poll_exit
2022-08-03 18:03:15,765 DEBUG   SenderThread:22252 [sender.py:send_request():250] send_request: poll_exit
2022-08-03 18:03:15,767 INFO    SenderThread:22252 [file_pusher.py:join():176] waiting for file pusher
2022-08-03 18:03:17,953 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: sampled_history
2022-08-03 18:03:17,954 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: get_summary
2022-08-03 18:03:17,956 DEBUG   HandlerThread:22252 [handler.py:handle_request():140] handle_request: shutdown
2022-08-03 18:03:17,956 INFO    HandlerThread:22252 [handler.py:finish():806] shutting down handler
2022-08-03 18:03:18,657 INFO    WriterThread:22252 [datastore.py:close():279] close: D:\my_code_2\my_code\tutorial-MixMatch-master\wandb\run-20220803_180040-mvzyeprd\run-mvzyeprd.wandb
2022-08-03 18:03:18,845 INFO    SenderThread:22252 [sender.py:finish():1114] shutting down sender
2022-08-03 18:03:18,846 INFO    SenderThread:22252 [file_pusher.py:finish():171] shutting down file pusher
2022-08-03 18:03:18,846 INFO    SenderThread:22252 [file_pusher.py:join():176] waiting for file pusher
2022-08-03 18:03:18,855 INFO    MainThread:22252 [internal.py:handle_exit():80] Internal process exited
